src/main/webapp/styles/style.css

```diff
@@ -65,4 +65,10 @@ header {
   text-align: center;
   padding: 10%;
   color: #fff;
+}
+
+/**************** HOME PAGE MAP *******************/
+#map {      	
```

Comment:
I would add this either to `index.css` or a new css file for the map. `style.css` is meant to apply to every single page we have, so it might mess things up later.